db/migrate/20230608002709_create_users.rb

```diff
@@ -2,6 +2,7 @@ class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
       t.string :name
+      t.string :email
```

Comment:
- [ ] I had issues setting up your database because you defined email for the users table, but in the migration file for installing Devise (`20230620142214_add_devise_to_users`) you have email again. Kindly remove this email here so the app uses the one from Devise.

  If you wish to reproduce this error you would need to drop your database and run your migrations afresh as a new user of your app would need to do.

  ![image](https://github.com/theodoku/blog/assets/36542986/1d5a5c76-68bf-4b6c-b8a8-4f338cc44fe5)